pkg/rotation/reconciler.go

```diff
@@ -88,6 +88,7 @@ func NewReconciler(s *runtime.Scheme, providerVolumePath, nodeName string, rotat
 	if err != nil {
 		return nil, err
 	}
+	config.UserAgent = "csi-secrets-store/rotation"
```

Comment:
adding version, hash, os and arch would be useful. you can pass version through ldflags

Reply:
https://github.com/open-policy-agent/gatekeeper/pull/858/files example

Reply:
Added it!